src/test/java/com/github/lwhite1/tablesaw/TestData.java

```diff
@@ -18,12 +15,19 @@
  */
 public enum TestData {
 
-    BUSH_APPROVAL(new ColumnType[]{LOCAL_DATE, INTEGER, CATEGORY}, "data/BushApproval.csv"),
```

Comment:
That delete is just the diffing tool being imperfect. 
Or do you mean I am not using this test data yet? I am sure it will be useful, but I wanted to start with something really simple since I needed to create known good results form the sort. The bushes and Tornados are still available as TestData though.